internal/commands/push/command.go

```diff
@@ -444,3 +444,40 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, remote appRemo
 	ui.Print(terminal.NewTextLog("Deployment complete"))
 	return nil
 }
+
+func importDependenciesAndWait(ui terminal.UI, realmClient realm.Client, remote appRemote, uploadPath string) error {
+	err := realmClient.ImportDependencies(remote.GroupID, remote.AppID, uploadPath)
+	if err != nil {
```

Comment:
[nit] can reduce the scope footprint of `err` by inlining the `if` statement here

also maybe it makes sense to include all of this within the "spinner scope", thoughts?